numpy/f2py/tests/util.py

```diff
@@ -30,6 +31,9 @@
 _module_dir = None
 _module_num = 5403
 
+if sys.platform == "cygwin":
+    _module_list = []
```

Comment:
If we want the tests to run with `python runtests.py`, rather than `pip install -e .; /usr/bin/rebase --database $(find numpy -name \*.dll); python runtests.py -n`, I'd want to add the rebase step in `runtests.py` and start this list with all the extension modules in numpy.

Reply:
The `runtests.py` workflow is going away with the move to meson, so the `pip install` workflow is fine.